test/functional/segwit.py

```diff
@@ -189,17 +189,13 @@ def run_test(self):
 
         # TODO: An old node would see these txs without witnesses and be able to mine them
 
-        self.log.info("Verify unsigned bare witness txs in versionbits-setting blocks are valid before the fork")
-        self.success_mine(self.nodes[2], wit_ids[NODE_2][WIT_V0][1], False) #block 428
-        self.success_mine(self.nodes[2], wit_ids[NODE_2][WIT_V1][1], False) #block 429
+        # TODO: verify spending a segwit v0 output is invalid before the fork
```

Comment:
In commit "[qa] Remove some pre-activation segwit tests"

Note: TODO

Reply:
I don't understand this comment. What does 'before the fork' mean if segwit is active since genesis?